edu_edfi_airflow/dags/edfi_resource_dag.py

```diff
@@ -358,21 +363,21 @@ def build_change_version_task_group(self) -> TaskGroup:
                 dag=self.dag
             )
 
-            # Reset the Snowflake change version table (if a full-refresh).
-            reset_snowflake_cvs = PythonOperator(
-                task_id="reset_previous_change_versions_in_snowflake",
+            # Reset the database change version table (if a full-refresh).
+            reset_database_cvs = PythonOperator(
+                task_id="reset_previous_change_versions_in_database",
```

Comment:
One thing I worry about this update is the loss of all prior logging after upgrading to use these new generic task names. I want us to discuss whether we should template the database type in this task_id or just rip off the bandaid to use the new format.

(Note that I'm addressing this in this comment alone, but this issue surfaces throughout the DAG.)